packages/amplify-category-notifications/package.json

```diff
@@ -29,7 +29,8 @@
     "inquirer": "^7.3.3",
     "lodash": "^4.17.21",
     "ora": "^4.0.3",
-    "promise-sequential": "^1.1.1"
+    "promise-sequential": "^1.1.1",
+    "chalk": "^4.1.1"
```

Comment:
if this was to fix a lint error for a missing dependency then it's okay. But we shouldn't be adding this dependency; instead us a method on the amplify printer

Reply:
it is both, there is some new code using chalk, and there was pre-existing one as well